src/app.js

```diff
@@ -110,7 +110,7 @@ export default function(app) {
       }
 
       for (var i in node.data) {
-        if (i === "oncreate") {
+        if (i == "oncreate") {
```

Comment:
Again, this _can_ fail if someone really wants to make it fail.

```
const a = {}
a.toString = () => 'oncreate'
a == 'oncreate' // true
```

Although this might actally be expected behaviour if you think about i.